package.json

```diff
@@ -134,5 +134,20 @@
         "dev": "node webpack/generateDetails.js && webpack --mode development --config webpack/webpack.dev.js --watch",
         "test": "node webpack/generateDetails.js && webpack --mode development --config webpack/webpack.dev.js",
         "build": "node webpack/generateDetails.js --prod && webpack --config webpack/webpack.prod.js"
+    },
+    "prettier": {
+        "plugins": ["prettier-plugin-jinja-template"],
+        "overrides": [
+            {
+                "files": [
+                    "*.html"
```

Comment:
The description says this changes how _Markdown lines are wrapped_, but this config appears to only apply to HTML files. Does the `proseWrap` option do something with Markdown in HTML? (I'm struggling to think of where we do that.) What am I missing?

Reply:
Currently we use Prettier for HTML files. This config tells Prettier to override the default parser for HTML files to use the "jinja-template" parser instead. These existing settings are unrelated to this PR, and the current HTML use case is unaffected by this PR.

This PR is to allow us to use Prettier for Markdown as well as HTML. [proseWrap](https://prettier.io/docs/options#prose-wrap) is the only setting that is relevant to this PR. The first commit deals with this Markdown use case.

The second commit moves Prettier's config -- the commit doesn't change the existing config.

I hope all that explains this PR a little better. :blowfish: Easier to review by commit.

Reply:
> This PR is to allow us to use Prettier for Markdown as well as HTML.

I'm not understanding how that works. Is it true that when the `proseWrap` setting is enabled, Prettier gains the ability to format Markdown files, and if it is not set it does nothing to them?

And what causes it to run on Markdown files? Is it something that needs to be done manually? Looks like our commit hook only applies to HTML and CSS files.

https://github.com/dimagi/commcare-hq/blob/cdc34b5a9e19bab696b8ccac43e5ea7e36fd5700/git-hooks/pre-commit.sh#L60-L62

Reply:
Got it, thanks for explaining. I didn't know our team was using `prettier` to format Markdown. Do you know if that's a workflow that many people are using?

Reply:
I discovered that `prettier` could format Markdown only recently, and I gave it a try. I don't know if anyone else is using it.

I will definitely be using it on my own Markdown changes going forward, and I'd encourage other people to use it too. It would be nice if we used it the way we use `isort`.